src/components/Actor/style.css

```diff
@@ -0,0 +1,6 @@
+.actor {
```

Comment:
jen pro úplnost, už to luděk na nějaké lekci říkal, ale jen si dej pozor na to že ty názvy class jsou globální, tj ty názvy by měly být unikátní skrz celý projekt. 

Dá se to udržet třeba tak že použiješ něco jako BEM názvy: {komponenta}__{třída}, tj v tomhle případě `.Actor__actor`

Tak jen FYI